scripts/create-aws-resources.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Convenience script for manually running create-aws-resources.py on app VM
+
+set -e
+set -x
+
+vagrant ssh app -c "cd /opt/app && envdir /etc/rf.d/env ./create-aws-resources.py"
```

Comment:
That would work for development, but I don't think test or production run the debugserver.